scripts/ci/lib.sh

```diff
@@ -1313,7 +1313,7 @@ __EOM__
       "$body")"
     echo -e "About to post:\n$payload"
 
-    echo "$payload" | curl -XPOST -d @- -H 'Content-Type: application/json' "$webhook_url" || {
+    echo "$payload" | curl -sSfl -XPOST -d @- -H 'Content-Type: application/json' "$webhook_url" || {
```

Comment:
Putting on my Malte hat, I would prefer to see these flags in full humanly understandable form:
`--?? --?? etc`

Reply:
```suggestion
    echo "$payload" | curl --location --silent --show-error --fail --data @- --header 'Content-Type: application/json' "$webhook_url" || {
```
`-XPOST` is not needed when `--data` is present